test_scripts/API/SetAudioStreamingIndicator/037_ATF_N_TC_SetAudioStreamingIndicator_USER_DISALLOWED_media_app_RPC_disallowed_by_user.lua

```diff
@@ -0,0 +1,190 @@
+config.application1.registerAppInterfaceParams.isMediaApplication = true
+
+--[[ Required Shared libraries ]]
+local commonFunctions = require ('user_modules/shared_testcases/commonFunctions')
+local commonSteps = require('user_modules/shared_testcases/commonSteps')
+local commonPreconditions = require('user_modules/shared_testcases/commonPreconditions')
+local testCasesForPolicyTable = require('user_modules/shared_testcases/testCasesForPolicyTable')
+local json = require('json')
+
+--[[ Local Functions ]]
+
+--[[@create_ptu_SetAudioStreamingIndicator_group: update preloaded_pt and creates PTU files
+! preloaded_pt.json: add Testing_group with RPC SetAudioStreamingIndicator and user consent
+! SetAudioStreamingIndicator_group1.json: Assign groups "Base-4", "Testing_group" to application
+! @parameters: NO
+]]
+local function create_ptu_SetAudioStreamingIndicator_group()
+  commonPreconditions:BackupFile("sdl_preloaded_pt.json")
+  local config_path = commonPreconditions:GetPathToSDL()
+
+  os.execute(" cp " .. config_path .. "sdl_preloaded_pt.json" .. " " .. "SetAudioStreamingIndicator_group1.json" )
+  local pathToFile = config.pathToSDL .. 'sdl_preloaded_pt.json'
+  local file = io.open(pathToFile, "r")
+  local json_data = file:read("*all")
+  file:close()
+
+  local data = json.decode(json_data)
+  if(data.policy_table.functional_groupings["DataConsent-2"]) then
+    data.policy_table.functional_groupings["DataConsent-2"].rpcs = json.null
+  end
+
+  data.policy_table.functional_groupings["Base-4"].rpcs.SetAudioStreamingIndicator = nil
+  data.policy_table.functional_groupings["Testing_group"] = {
+    user_consent_prompt = "Purpose_of_Test",
+    rpcs = {
+      SetAudioStreamingIndicator = {
+        hmi_levels = { "BACKGROUND", "FULL", "LIMITED" }
+      }
+    }
+  }
+  local data_preloaded = commonFunctions:cloneTable(data)
+
+  data_preloaded = json.encode(data_preloaded)
+  file = io.open(config_path .. 'sdl_preloaded_pt.json', "w")
+  file:write(data_preloaded)
+  file:close()
+
+  data.policy_table.module_config.preloaded_pt = nil
+  data.policy_table.module_config.preloaded_date = nil
+  data.policy_table.app_policies[config.application1.registerAppInterfaceParams.appID] =
+  {
+    keep_context = false,
+    steal_focus = false,
+    priority = "NONE",
+    default_hmi = "NONE",
+    groups = {"Base-4", "Testing_group"}
+  }
+
+  data = json.encode(data)
+  file = io.open("SetAudioStreamingIndicator_group1.json", "w")
+  file:write(data)
+  file:close()
+end
+
+
+--[[ General Precondition before ATF start ]]
+create_ptu_SetAudioStreamingIndicator_group()
+commonSteps:DeleteLogsFiles()
+commonSteps:DeletePolicyTable()
+
+--[[ General Settings for configuration ]]
+Test = require('connecttest')
+require('user_modules/AppTypes')
+
+--[[ Preconditions ]]
+commonFunctions:newTestCasesGroup("Preconditions")
+
+function Test:Precondition_trigger_getting_device_consent()
```

Comment:
@istoimenova, Getting device consent is not required for PROPRIETARY flow to perform PTU